Modify S6105: Move to progressive education format (APPSEC-422) (#1519)
SonarSource/rspec: rules/S6105/rationale.adoc

> Open redirection occurs when an application uses user-controllable data to redirect users to a URL.

Reviewer: ```suggestion
Open redirection occurs when an application uses user-controllable data to build URLs used during redirects.
```
